registration/backends/admin_approval/urls.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
I am not sure whether this is possible after all. 

The urls in ``admin_approval/urls.py`` use different views than the ones in ``default/urls.py``. The difference is that the first ones use the new ``SupervisedRegistrationProfile``. I am not sure how to override this or if it is worth doing so. What do you think?